cogs/welcome/welcome.py

```diff
@@ -28,8 +31,22 @@ class Welcome(commands.Cog):  # pylint: disable=too-many-instance-attributes
     def __init__(self, bot: Red):
         self.bot = bot
         self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+
         self.config.register_guild(**DEFAULT_GUILD)
 
+        self.dataDir = data_manager.cog_data_path(cog_instance=self)
+        self.imgDir = self.dataDir / WELCOME_IMG_FOLDER
+        self.bundled_assets = data_manager.bundled_data_path(self)
+
+        # create folder to hold welcome images
+        try:
+            os.makedirs(self.imgDir, exist_ok=True)
+        except OSError as error:
+            LOGGER.error(
+                "Could not create folder for images!",
+                exc_info=True,
+            )
```

Comment:
This seems pretty bad of an error, can we please re-raise the exception so that we're notified when we try to load the cog?